src/image-default.yaml

```diff
@@ -10,6 +10,8 @@ extra-kargs: []
 
 # Can also be oci-chunked
 ostree-format: oci-chunked-v1
+# Inject io.openshift.build.version-display-names for OpenShift CVO
+ostree-container-inject-openshift-cvo-labels: false
```

Comment:
I wonder if we should break this down into something more generic, like `ostree-container-labels: []` or something (with a `BUILDID` template allowed). That way those special labels live in `openshift/os` directly.

Cool as is too though!

Reply:
I'm uncertain in what scenario we'd want such programmability versus defaulting to do it in coreos-assembler code.

Personally, I also see this as a short term hack.  The right fix I think is to actually teach `oc adm release new` about operating system images more natively.

That said again, actually, it could make sense to have some of the metadata from `os-release` injected into the container image metadata by default; perhaps ostree code should do that automatically and by default.

Reply:
Stuck this thought over in https://github.com/openshift/os/issues/1047